### PR TITLE
docs(website): add simplifed chinese

### PR DESCRIPTION
**Pull request by zhonger**
# Summary

Docs(website): Add simplified chinese

### CODE REVIEW

File: docs/contribute/core.md [hunk 1]
```diff
@@ -94,10 +94,10 @@ feat!: feature with breaking change
...
 The full list of `<types>` are: `feat`, `fix`, `docs`, `style`, `refactor`, `perf`, `test`, `build`, `ci`, `chore`, `revert`.
 
-* `!`: indicates a breaking change
-* `fix`: will create a new SemVer `patch`
-* `feat`: will create a new SemVer `minor`
-* `<type>!`: will create a new SemVer `major`
+- `!`: indicates a breaking change
+- `fix`: will create a new SemVer `patch`
+- `feat`: will create a new SemVer `minor`
+- `<type>!`: will create a new SemVer `major`
```
Comment (Stratus3D) [inline docs/contribute/core.md:100]: Why were these changed?

**Review by Stratus3D — APPROVED** [review]
This is great! I can't read chinese so I can't say if everything here is correct, but scanning through it all I don't see any obvious issues with the changes. Thanks for the PR @zhonger !

**Comment by Stratus3D** [comment]
@asdf-vm/core can anyone else take a look at these changes? I see no reason not to merge this now, so we are currently lacking a chinese translation of the docs.

File: docs/contribute/core.md (hunk 1 above)
Comment (zhonger) [reply docs/contribute/core.md:100]: I am so sorry for that. It should be not changed. 
In order to distinguish which is changed by me and the `format` tool, I have made two commits.
We can see easily that this change is because of the `format` tool.

**Comment by Stratus3D** [comment]
Since these changes are mostly additive I'm going to go ahead and merge this. I tried google translate on some of the docs and validated that the text in the docs translates to roughly the same thing as what we have for the english docs, but I would love another review from someone who can read the language.
